internal/cba.go

```diff
@@ -67,6 +74,15 @@ func getClientCertificateSourceAndEndpoint(settings *DialSettings) (cert.Source,
 	if err != nil {
 		return nil, "", err
 	}
+	// TODO(chrisdsmith): Use this composed endpoint everywhere to replace DialSettings.DefaultEndpoint
```

Comment:
nit: instead of tagging these as todo to you can you file issues for this and link to the issue number

Reply:
Done, thanks!